builds/e2e/connectivity.yaml

```diff
@@ -169,6 +168,46 @@ jobs:
             $(images.artifact.name.linux)/scripts/local/test/DownloadIdentityService.ps1
             $(images.artifact.name.linux)/CACertificates/certGen.sh
             $(images.artifact.name.linux)/CACertificates/openssl_root_ca.cnf
+      - task: AzureCLI@2
+        displayName: 'Get SAS URI for Blob Storage Account'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          scriptType: 'bash'
+          scriptLocation: 'inlineScript'
+          inlineScript: |
+            # Hardcoded resource group and storage account name
+            resource_group="iotedge-deploy"
+            storage_account="iotedgeconnectivitytest"
```

Comment:
Please make these task variables.

Reply:
@yophilav task variables, as in, `##vso[task.setvariable ...]`? These are only needed within the bash script, so bash vars are fine.